app/_hub/kong-inc/standard-webhooks/overview/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+nav_title: Overview
+title: Overview
+---
+
+The Standard Webhooks plugin lets you validate incoming webhooks using the [Standard Webhooks](https://github.com/standard-webhooks/standard-webhooks) specification. 
+Kong is a contributor to this specification.
+
```

Comment:
The Standard Webhooks specification is a package of open source tools and guidelines for sending webhooks easily, securely, and reliably.